docs/src/tutorial/example-data.rst

```diff
@@ -54,7 +54,7 @@ The workflow can take several minutes to run. While it is running, you can learn
    - This required Snakemake parameter specifies the number of CPU cores to use (:doc:`more info <snakemake:executing/cli>`).
 - ``--configfile ncov-tutorial/example-data.yaml``
    - ``--configfile`` is another Snakemake parameter used to configure the ncov workflow.
-   - ``ncov-tutorial/example-data.yaml`` is a YAML file which provides custom workflow configuration including inputs and outputs. Contents with comments excluded:
+   - ``ncov-tutorial/example-data.yaml`` is a YAML file that provides custom workflow configuration including inputs and outputs. The contents of this file with comments excluded are:
```

Comment:
YAML detail can be kept in [the glossary entry](https://nextstrain--942.org.readthedocs.build/projects/ncov/en/942/reference/glossary.html#term-config-file).

```suggestion
   - ``ncov-tutorial/example-data.yaml`` is a configuration file that provides custom workflow configuration including inputs and outputs. The contents of this file with comments excluded are:
```